lib/svg-sprite/config.js

```diff
@@ -145,7 +145,8 @@ function SVGSpriterConfig(config) {
             meta = meta ? yaml.load(meta) : {};
             for (const m in meta) {
                 if (_.isPlainObject(meta[m])) {
-                    this.shape.meta[path.join(path.dirname(m), path.basename(m, '.svg'))] = _.pick(meta[m], ['title', 'description']);
+                    const { title, description } = meta[m];
+                    this.shape.meta[path.join(path.dirname(m), path.basename(m, '.svg'))] = { title, description };
```

Comment:
Now that I see this, maybe we could skip the new var and directly access the keys. Let me know what you prefer

Reply:
I think it's ok here. But I personally prefer skipping such variables. In this case I see no problem

Reply:
My only concern is if we end up with undefined `title` and/or `description`. If so, the new code will probably throw.